setup.py

```diff
@@ -63,6 +63,7 @@ def get_long_description():
     "tqdm",
     "gevent>=20",
     "datahugger>=0.2",
+    "tomli",  # included in Python 3.11 as tomllib
```

Comment:
See the syntax below for importlib_metadata.

Reply:
Done.

Reply:
But where?